app/src/cli/cli.cpp

```diff
@@ -487,13 +488,8 @@ void parseRecover(const std::vector<std::string> &, ArgumentData &argument_data)
     argument_data.tournament_config.recover = true;
 }
 
-void parseRandomSeed(const std::vector<std::string> &, ArgumentData &argument_data) {
-    argument_data.tournament_config.randomseed = true;
-
-    std::random_device rd;
-    std::mt19937_64 gen((static_cast<uint64_t>(rd()) << 32) | rd());
-    std::uniform_int_distribution<uint64_t> dist(0, std::numeric_limits<uint64_t>::max());
-    argument_data.tournament_config.seed = dist(gen);
+void parseRandomSeed(const std::vector<std::string> &, ArgumentData &) {
```

Comment:
I think this can now be removed, together with the line that calls this function?

Reply:
I'd rather keep it there so if someone specifies '-randomseed' it will just be silently ignored instead of throwing an error

Reply:
I think this PR has moved to mirror cute-chess, which I think is good. So then we should also remove this. It's a simple change for any user getting caught out.

Reply:
removed for now, but might add a logger warning in the future that the option has been removed if someone asks why the option throws an error